app/views/works/index.html.erb

```diff
@@ -0,0 +1,111 @@
+<section class="flash">
+  <p> <%= flash[:success] %></p>
+</section>
+
+<main>
+
+  <h2>List of Works</h2>
+
+  <section class="media-table">
+    <h4>Albums</h4>
+
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Votes</th>
+          <th>Title</th>
+          <th>Created By</th>
+          <th>Published</th>
+          <th>Upvote</th>
+        </tr>
+      </thead>
+      <tbody>
+
+        <% albums = Work.sort_album_works%>
+
+        <% albums.each do |album| %>
+
+          <tr>   
+            <td><%= album.votes.count %></td>
+            <td><%= link_to album.title, work_path(album.id) %></td>
+            <td><%= album.creator %></td>
+            <td><%= album.publication_year %></td>
+            <td><%= link_to "Upvote", work_votes_path(album.id), method: :post, class: "btn btn-primary" %></td>
+          </tr>
+
+        <% end%>
+        
+      </tbody>
+    </table>
+  </section>
+
+
+
+  <section class="media-table">
+    <h4>Books</h4>
+
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Votes</th>
+          <th>Title</th>
+          <th>Created By</th>
+          <th>Published</th>
+          <th>Upvote</th>
+        </tr>
+      </thead>
+      <tbody>
+
+        <% books = Work.sort_book_works%>
+
+        <% books.each do |book| %>
+
+          <tr>   
+            <td><%= book.votes.count %></td>
+            <td><%= link_to book.title, work_path(book.id) %></td>
+            <td><%= book.creator %></td>
+            <td><%= book.publication_year %></td>
+            <td><%= link_to "Upvote", work_votes_path(book.id), method: :post, class: "btn btn-primary" %></td>
+          </tr>
+
+        <% end%>
+        
+      </tbody>
+    </table>
+  </section>
+
+
+  <section class="media-table">
+    <h4>Movies</h4>
+
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Votes</th>
+          <th>Title</th>
+          <th>Created By</th>
+          <th>Published</th>
+          <th>Upvote</th>
+        </tr>
+      </thead>
+      <tbody>
+
+        <% movies = Work.sort_movie_works%>
+
+        <% movies.each do |movie| %>
```

Comment:
Just noting that these things could be done with a set of partial views.  You could build this view and use the same partial view 3 times.